### DISCUSSION TITLE
The bridge is working in one way only

### DISCUSSION THREAD

**Discussion by BornToChop** [wormhole-foundation/wormhole#206] [Problems / bugs] [answered]
Hello,

I transfered $aleph (erc-20) tokens to solana blockchain through Wormhole, 2 days ago.
![wormhole](https://user-images.githubusercontent.com/64804728/120533043-9c75f980-c3e0-11eb-967f-46b0a7e90ea9.jpg)
 However I was not able to do anything with tokens I received in return (neither exchange nor farm on raydium
![sollet](https://user-images.githubusercontent.com/64804728/120533655-405fa500-c3e1-11eb-9747-cca91320efdb.png)
). Therefore I tried to transfer them back on ethereum... But it turned out it is impossible. Firstly, Wormhole sees only 0.00001593 $aleph in my sollet while solana explorer shows 15 931 (this is by the way the number of tokens I had on ethereum network)
![wormhole transfer](https://user-images.githubusercontent.com/64804728/120533858-7ac94200-c3e1-11eb-8844-7ce2bc48e9fd.png)
. Secondly, each time I'm trying to exchange 0.000015931 tokens that Wormhole recognizes, transaction fails. 
![fails](https://user-images.githubusercontent.com/64804728/120533959-9896a700-c3e1-11eb-8da3-9d53d8c832d8.png)

So I wonder, if there is an option of manual exchange. If no, how could I get my erc-20 tokens back please?

Here is my ethereum wallet : https://etherscan.io/address/0x0d9686d3c602a458110f43147bcf33bfc3fef135
Here is my solana wallet : https://explorer.solana.com/address/4wyfVQ6w2Lnv3cdBc4dGpweaWZ7J8UzixWGLgRMX47SK/tokens

Kind regards

**Comment by BornToChop** [comment]
Still not working
![image](https://user-images.githubusercontent.com/64804728/121064904-7083bb00-c7c8-11eb-8351-82cf71d18c8a.png)

**Comment by hendrikhofstadt** [comment, marked as answer]
We don't support the UI. It was built by a third party and unfortunately we can't help with issues there.

**Comment by leoluk** [comment]
You can create issues for wormholebridge.com here: https://github.com/solana-labs/oyster/issues

**Comment by leoluk** [comment]
Did this get resolved? I saw that wormholebridge.com recently added a bunch of new tokens, was yours among them?
